### ISSUE TITLE
extractor.yml.sample - max_is_error

### ISSUE THREAD

**Issue by kurgans0** [pandora-analysis/pandora#163]
On file extractor.yml when the setting "max_is_error" has value "**no**", Pandora generates an "alert" status (if the zip file contains more than "max_files_in_archive").
But if it's "**yes**", Pandora has a "clean" status (with a warning message).

1/ It should be the opposite, isn't it?

2/ When "alert" status isn't wanted, is it possible to have the choice to generate "warn" or "clean" status using a new settings?

**Comment by Rafiot** [comment]
Yeah, I don't like this feature (it should be left as it is by default, to `no`), but it was because someone wanted to be able to treat the case a an archive has too many files (or a too big one) as an error instead of a suspicious behavior. 

When a worker returns an error, it isn't considered suspicious and won't trigger the file to be considered suspicious. Generally error is raised when a 3rd party service isn't responding.

I do not consider this feature to be a good idea and I'd always recommend to raise an alert when we reach the any of the max (number of file or size), and increase that limit as much as needed.

**Comment by kurgans0** [comment]
No pb. Thanks to have take time to answer about point 2/.

**Comment by Rafiot** [comment]
Oh, sorry, totally missed it.

The easiest is probably to disable the module all together (for that, you an remove the .yml file in the workers directory): a worker is a unit and arbitrarily changing their output status sounds pretty dangerous to me (it's already a bit of a hack for the extractor module).
Do you have a practical use-case for that?

**Comment by Rafiot** [comment]
Right I understand, and it makes sense... Is now you're mentioning it, I'm unsure why we opted for ALERT instead of WARN. The goal is to make sure the users go talk to someone before opening any file that isn't CLEAN, and WARN should be enough in that case.
